.travis.yml

```diff
@@ -1,11 +1,29 @@
----
 language: ruby
 sudo: false
 rvm:
-  - 2.2
+  - 2.3.0
+  - 2.2.4
   - 2.1
-  - 2.0.0
-  - 1.9.3
-  - jruby-19mode
-before_script: 'bundle exec appraisal install'
-script: 'bundle exec rake appraisal test'
+  - 2.0
+  - 1.9
+  - jruby-9.0.1.0
```

Comment:
This is 1.9 mode, and hence fails for Rails 5 right now(Specified this to pick the latest JRuby)
2.2 mode is not out yet on travis..